### PR TITLE
Update changelog for 1.80.0

### PR DESCRIPTION
**Pull request by Veykril**

### CODE REVIEW

File: src/values.rst [hunk 1]
```diff
@@ -37,6 +48,20 @@ Two :t:`[value]s` :t:`overlap` when
...
+
+:dp:`fls_uhwpuv6cx4ip`
+It is undefined behavior to create an :t:`allocated object` with :t:`memory
+size` ``size`` at a :t:`base address` ``base`` where ``base + size`` is greater
+than the architectures maximum :c:`usize` value.
+
+:dp:`fls_xuuFKmm181bs`
+It is undefined behavior to create an :t:`allocated object` with :t:`memory
+size` ``size`` where ``size`` is greater than the architectures maximum
+:c:`isize` value.
```
Comment (pietroalbini) [inline src/values.rst:63]: This should be in an undefined behavior rubric, so that it shows up in the UB appendix.

**Review by pietroalbini — CHANGES_REQUESTED** [review]

File: src/values.rst (hunk 1 above)
Comment (Veykril) [reply src/values.rst:63]: It already is (see line 45)

Comment (pietroalbini) [reply src/values.rst:63]: Whoops!

**Review by pietroalbini — APPROVED** [review]
bors merge

**Pull request merged**